Mixpanel/InAppNotification.swift

```diff
@@ -55,7 +57,23 @@ class InAppNotification {
         }
 
         guard let extras = object[PayloadKey.extras] as? [String: Any] else {
-            Logger.error(message: "invalid notification extra section")
+            Logger.error(message: "invalid notification extra section \(object)")
+            return nil
+        }
+        
+        if let rawDisplayTriggers = object[PayloadKey.displayTriggers] as? [[String: Any]]? {
+            if let rawDisplayTriggers = rawDisplayTriggers {
```

Comment:
is it redundant here for assigning `rawDisplayTriggers` again?

Reply:
rawDisplayTriggers is optional let me rename it to make it obvious